src/dmd/cli.d

```diff
@@ -352,11 +352,21 @@ dmd -cov -unittest myprog.d
                 $(LINK2 http://dlang.org/windbg.html, Debugging on Windows).
             )
             $(UNIX
-                Add symbolic debug info in Dwarf format
+                Add symbolic debug info in DWARF format
                 for debuggers such as
                 $(D gdb)
             )`,
         ),
+        Option("gdwarf=<version>",
+            "add symbolic debug info in a specified version of the DWARF format",
+            "Versions supported are:
+            $(UL
+                $(LI $(I 3) (default): emit DWARF v3 debug informations)
+                $(LI $(I 4): emit DWARF v4 debug informations)
+                $(LI $(I 5): emit DWARF v5 debug informations)
```

Comment:
Version 5 is not listed as supported in the code.

Reply:
Does this really need to be listed?  This could be abbreviated as `The minimum supported version number is 3`, which leaves it open ended for version 6, 7, 8, and beyond..

Reply:
As end-user, I would like to know which DWARF versions are supported. Don't you agree ?

Reply:
It could still be as a sentence: `The value of version may be either 3, 4 or 5. The default version is 3.`